### DISCUSSION TITLE
Using the Delay component

### DISCUSSION THREAD

**Discussion by prlaba** [hneemann/Digital#1122] [General]
The Delay component seems a good way to roughly simulate propagation delays within a circuit.

In my case, I have circuits consisting of typical logic gates, multiplexers, decoders, etc.  And some very slow EEPROMs.  EEPROMs have access times in the 90-150 nsec range, compared to most components with much smaller access times (5-20 nsecs).

The Delay component allows you set its duration (delay time) between 1 and 5.  Can you increase the maximum to 10 or 15, to account for very slow components?

**Comment by hneemann** [comment]
The maximum value is 20, just enter 15.

**Reply by prlaba** [reply]
Hmm... I've gotten tripped up by this before.  The drop-down list for the Delay component lists the values 1 to 5.  It offers no clue that a larger value is allowed, or what the maximum value is.  In fact it led me to believe the maximum value is 5.

Perhaps you could add the values 10, 15 and 20 to the list?  That might make it more clear that values up to 20 are allowed.
